metals/src/main/scala/scala/meta/internal/metals/BloopInstall.scala

```diff
@@ -243,9 +244,14 @@ object BloopInstall {
         fn: String => Unit
     ): Unit = {
       if (!closed) {
-        val text = toPlainString(buffer).trim
-        if (text.nonEmpty) {
-          fn(text)
+        val text = toPlainString(buffer)
+        output = output + text
```

Comment:
isn't it équiva and then Java lent of appending inside a loop? Here, we are not the one controlling how often will that happen, so (even though Java 8 and then Java 9 greatly increased the performance of appending strings) I am not sure just concatenating strings here is the best approach.

Reply:
Using StringBuilder after sugestion